python/simulators/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+pybind11_add_module(simulators simulators.cpp)
+
+set_target_properties( simulators PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/python/opm2 )
+
+target_sources(simulators
+  PRIVATE
+    ../../flow/flow_ebos_blackoil.cpp
+    ../../flow/flow_ebos_brine.cpp
```

Comment:
Note: It might be desirable to remove the dependencies on the other source files other than that of the blackoil simulator `flow_ebos_blackoil.cpp` . This will reduce compile time for the Python module, and also make the library smaller. Currently, the Python bindings does not use any of  the sources: `flow_ebos_brine.cpp`, `flow_ebos_energy.cpp`, `flow_ebos_foam.cpp`, `flow_ebos_gasoil.cpp`, `flow_ebos_oilwater.cpp`, `flow_ebos_oilwater_polymer.cpp`, `flow_ebos_oilwater_polymer_injectivity.cpp`, `flow_ebos_polymer.cpp`, `flow_ebos_solvent.cpp`.. But we still need to compile them into the Python library since they are mentioned in the new `Main.hpp`, see PR #2516

Reply:
In order to do that, I think you may have to define FLOW_BLACKOIL_ONLY in simulators.cpp, otherwise you will fail to link to the other variants.

The C++ parts of this looks fine to me. I am no Python expert, someone else must decide if those parts are good.

Reply:
> In order to do that, I think you may have to define FLOW_BLACKOIL_ONLY in simulators.cpp,

Yes, of course! I pushed an update, but for some reason it does not show here. It is available in my fork though: https://github.com/hakonhagland/opm-simulators-1/commit/b2ec7cd9519cf337a07afcde5a25c4d2018c2ffe